pages/Configuring/Configuring-Hyprland.md

```diff
@@ -6,20 +6,21 @@ autogenerated config.
 By removing the line `autogenerated=1` you'll remove the yellow warning.
 
 There is no "reload" keybind. The config is reloaded the moment you save it.
+However, some options may require a restart before taking effect.
 
-Start a section with `name {` and end in `}` ***in separate lines!***
+Start a section with `name {` and end in `}` **_in separate lines!_**
 
 {{< hint type=important >}}
-The default config is *very* barebones. Please refer to this wiki page and the pages
+The default config is _very_ barebones. Please refer to this wiki page and the pages
 linked further down here for full configuration instructions.
 
 **Make sure to read the "Variables" page as well**. It covers all the
 toggleable / numerical options.
 {{< /hint >}}
 
-# Line style
+## Line style
```

Comment:
why

Reply:
Because we want to follow basic markdown rules? If we start here with a h1 title and in other pages h2, it's just not a good thing to have.

Reply:
Make it start with h1 on all. ToC looks better that way